packages/material-ui-icons/builder.js

```diff
@@ -14,6 +14,25 @@ const globAsync = util.promisify(glob);
 export const RENAME_FILTER_DEFAULT = './renameFilters/default';
 export const RENAME_FILTER_MUI = './renameFilters/material-design-icons';
 
+let blacklistIcons = [
+  '6FtApart', // Arbitrary covid related distance
```

Comment:
👀 (Interesting how many covid related icons they have already, I guess it’s been 5 months)

Reply:
I have removed this one because:

1. ![w](https://cdn.statcdn.com/Infographic/images/normal/18300.jpeg)
2. before we know it, COVID-19 will be behind us, 2 years top (either with vaccine or herd immunity?) Take France, we are 6 months in, 10% of the population is estimated to have caught the virus. If herd immunity is at 70%, we need x7 more months to reach it assuming linear growth (what government aims to) 1.75 years.